templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_pageheader.html.twig

```diff
@@ -74,7 +74,7 @@
                 <li class="inline-block {{ loop.last ?: 'u-mr-0_5' }}">
                     <a
                         class="{{ item.current|default == true  ? 'o-link--active' : 'o-link--default' }} whitespace-nowrap"
-                        :data-cy="`pageHeader:item:{{ loop.index }}`"
+                        :data-cy="`pageHeader:item:{{ item.datacy }}`"
```

Comment:
What happens if `datacy` is not defined?

Reply:
then i would be undefined or null :D but I hope it won’t happen, because they always need the same naming for testing. If I choose a different one as the default value, I think it would not make sense, because tests will crash

Reply:
but there are usages of `base_pageheader` with `subnav` without `datacy` defined for the item(s), so I think we should adjust these usages

Reply:
ach, got you! I was too focused on other things :D You are totally right, there are some usages without datacy, so it needs to be adjusted and will take more time. I will do it a bit later. thx for spotting it!

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/4510/commits/6cb8042e3b2efbbae34393db3c2f965225e6a126